File: python Important question/GreaterThreeNumber.py
# Write a program to find the greatest number among three number.
number1 = int(input("Enter the number 1: "))
number2 = int(input("Enter the number 2: "))
number3 = int(input("Enter the number 3: "))

if(number1>0 and number2>0 and number3>0):
 if(number1 > number2 and number1 > number3):
    print("Number 1 is greater")
 elif(number2 > number1 and number2 > number1):
    print("Number 2 is greater")
 elif(number3 > number1 and number3 > number2):
    print("Number 3 is greater")
 else:
    print("All are equal")

File: Java im portant question/GreaterThreeNumber.java
/*
Write a program to find the greatest number among three number.
 */
import java.util.Scanner;
public class GreaterThreeNumber
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter number1: ");
        int number1 = sc.nextInt();
        System.out.println("Enter number2: ");
        int number2 = sc.nextInt();
        System.out.println("Enter number3: ");
        int number3 = sc.nextInt();
        int max = number1;

        if (number2 >= max){
            max = number2;
        }
        if (number3 >= max) {
            max = number3;
        }
        System.out.println("Greater number is "+ max);
    }
}
